pkg/codegen/testing/test/program_driver.go

```diff
@@ -381,7 +381,10 @@ func TestProgramCodegen(
 	for _, tt := range testcase.TestCases {
 		tt := tt // avoid capturing loop variable
 		t.Run(tt.Directory, func(t *testing.T) {
-			t.Parallel()
```

Comment:
Will this cause a lint failure in `paralleltest`?

Reply:
@abhinav huh, it looks like it didn't. How did this escape `paralleltest`?

Reply:
I'm guessing that paralleltest doesn't care about this because this isn't a _test.go file or a test function (`func TestFoo(t *testing.T)`). I left that comment in there in case paralleltest got smarter in the future.